### ISSUE TITLE
Left-hand nav sidebar changes size in admin panel

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#14747]
When I click Enable Caching, it changes the size of the left-hand nav sidebar. I guess it’s just matching the content in the center of the page, but it’s visually jarring. Is this intentional?

![sidebar](https://user-images.githubusercontent.com/1455846/107564487-57ab2600-6b97-11eb-898e-54586ad97d0a.gif)

**Comment by camsaul** [comment]
@mazameli I'm not sure if this is actually a bug or not. Is this intentional?

**Comment by mazameli** [comment]
Yeah, recategorizing as UX. That side panel is supposed to grow to match the height of the content, but it does seem a little funny looking in your GIF here.

**Comment by brunobergher** [comment]
Should be handled by #41189
